### PR TITLE
adding works

### PR DESCRIPTION
**Pull request by Kuba-Wi**

### CODE REVIEW

File: module1/homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -2,6 +2,22 @@
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+     int result;
```
Comment (adamvm) [inline module1/homework/calculate/calculate.hpp:5]: some formatting issue

Comment (adamvm) [reply module1/homework/calculate/calculate.hpp:5]: one space in front of

Comment (Kuba-Wi) [reply module1/homework/calculate/calculate.hpp:5]: ok

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1429, 1119, 1430], "points": [2, 5, 2, 6] }

**Pull request closed without merging**